qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -97,7 +97,7 @@ def install_deps(self):
 
         distro = get_system_type(self.mount_a.client_remote,
                                  distro=True).lower()
-        if distro in ['redhatenterpriseserver', 'fedora', 'centos']:
+        if distro in ('redhatenterpriseserver', 'RedHatEnterprise', 'fedora', 'centos'):
```

Comment:
Captializing shouldn't be necessary, but, yeah, that makes it more readable.

Reply:
you're right, fixed!